infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto

```diff
@@ -88,11 +88,16 @@ message KaspadMessage {
 // P2P                                                                     //
 /////////////////////////////////////////////////////////////////////////////
 
-// AddressesMessage start
-message AddressesMessage{
+// GetAddressesMessage start
```

Comment:
Comment says GetAddressesMessage while message is called RequestAddressesMessage